topics/galaxy-interface/tutorials/group-tags/tutorial.md

```diff
@@ -99,9 +99,8 @@ The second method works at any step during the analysis.
 >
 >    - Click on **Rules** and then **Add / Modify Column Definitions**
 >    - Click on **Add Definition** and select:
->      - *"URL"*: Column A
```

Comment:
I see the option to make A the URL (it should be absolutely necessary, otherwise how would the data get imported?) I don't think we should remove that.

![image](https://user-images.githubusercontent.com/458683/116053329-daae2980-a67a-11eb-8494-d9d4a761ab9a.png)

Reply:
True. Seems that I was "abusing" the tutorial and just applied the `Apply rule to collection` tool. But this is also suggested in the intro:  _There are three ways to set group tags: Using the rule builder / apply_rules tool ..._ Maybe we should re-formulate such that its clear that the URL option is unavailable to the `Apply rule` tool.

But in the `Apply rule` tool quite a lot of options are missing (certainly URL seems not needed in this context). I remember that there was an issue but can't find it. 

Given that group tags do not propagate at the moment (https://github.com/galaxyproject/galaxy/issues/8554) I find the `Apply rule to collection` more applicable in real world scenarios.

Reply:
Ahhh that makes sense then. Yes, we definitely need to clarify that.

And, agreed @ lack of propagation meaning doing it in upload is often less useful. We should update the tutorial to include a more relevant example.

Reply:
I've made a change to the intro to clarify. Are you ok with that change?